frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -214,6 +228,7 @@ function ReaderDictionary:addToMainMenu(menu_items)
         text = _("Dictionary settings"),
         sub_item_table = {
             {
+                keep_menu_open = true,
```

Comment:
May be the menu item text below should be:
```diff
- return T(_("Installed dictionaries (%1)"), nb_str)
+ return T(_("Managed dictionaries (%1)"), nb_str)
```

Reply:
What about `Manage dictionaries (%1)`, I think this fits better to the following `Download dictionaries` menu option.